tools/update-tester/src/installer.rs

```diff
@@ -93,17 +93,17 @@ fn get_current_checkout() -> xshell::Result<String> {
     cmd!("git rev-parse --verify HEAD").read()
 }
 
-fn get_pgx_version(cargo_toml_contents: &str) -> Version {
+fn get_pgrx_version(cargo_toml_contents: &str) -> Version {
     let cargo = cargo_toml_contents
         .parse::<Document>()
         .expect("invalid Cargo.toml");
 
-    cargo["dependencies"]["pgx"]
+    cargo["dependencies"]["pgrx"]
         .as_str()
-        .expect("expected pgx to only have a version")
+        .expect("expected pgrx to only have a version")
         .trim_start_matches(['=', '^', '~'].as_slice())
         .parse()
-        .expect("cannot parse pgx version")
+        .expect("cannot parse pgrx version")
 }
```

Comment:
I think we might run into some issues with the update-tester with these changes. After checking out a git tag, get_pgrx_version() will read the new Cargo.toml from the filesystem, which will still be listed as 'pgx' for all but this newest version which I think will cause a panic.